packages/react/src/publication/adapters/ReactionPresenter.ts

```diff
@@ -50,6 +55,33 @@ export class ReactionPresenter implements IReactionPresenter {
     });
   }
 
+  async revert(request: ReactionRequest): Promise<void> {
```

Comment:
@desfero what do you think about reverting a reaction to the previous state in case of error?